src/java.base/share/classes/java/lang/Long.java

```diff
@@ -1245,8 +1245,8 @@ public static int hashCode(long value) {
      *          {@code false} otherwise.
      */
     public boolean equals(Object obj) {
-        if (obj instanceof Long) {
-            return value == ((Long)obj).longValue();
+        if (obj instanceof Long ell) {
+            return value == ell.longValue();
```

Comment:
```suggestion
        if (obj instanceof Long l) {
            return value == l.longValue();
```

Reply:
In other place in the code, I've written out "ell" to avoid any confusion between lowercase "l" and a number "1", etc.

Reply:
What does "ell" stand for? Looks like "eel" to me.
That aside, if there's no good name, we usually use `that` or `o` for the casted variable, like
```
return obj instanceof Long that && this.value == that.value;
```

Reply:
"ell" is a way to write out the name for the letter "L" phonetically.